dotCMS/src/main/java/com/dotmarketing/portlets/cmsmaintenance/ajax/CMSMaintenanceAjax.java

```diff
@@ -376,4 +428,293 @@ public void zipTempDirectoryToStream(OutputStream out) throws IOException {
+					}
+					else if(LinkVersionInfo.class.equals(clazz)){
+						_dh.setSQLQuery("SELECT {link_version_info.*} from link_version_info link_version_info, identifier where identifier.id = link_version_info.identifier order by link_version_info.identifier ");
+					}
+					else if(CalendarReminder.class.equals(clazz)){
+						_dh.setQuery("from " + clazz.getName() + " order by user_id, event_id, send_date");
+					} else if(Identifier.class.equals(clazz)){
+						_dh.setQuery("from " + clazz.getName() + " order by parent_path, id");
+					} else {
+						_dh.setQuery("from " + clazz.getName() + " order by 1");
+
+					}
+
+					_list = _dh.list();
+					if(_list.size() ==0){
+						try {
+							_bout.close();
+						}
+						catch( java.lang.NullPointerException npe){}
+						_list = null;
+						_dh = null;
+						_bout = null;
+
+						break;
+					}
+
+					if(_list != null && _list.size() > 0 && _list.get(0) instanceof Comparable){
+						java.util.Collections.sort(_list);
+					}
+
+					_writing = new File(backupTempFilePath + File.separator +  clazz.getName() + "_" + formatter.format(i) + ".xml");
+					_bout = new BufferedOutputStream(Files.newOutputStream(_writing.toPath()));
+
+					total = total + _list.size();
+
+					try {
+						Thread.sleep(10);
+					} catch (InterruptedException e) {
+						Logger.warn(this, "An error ocurred trying to create XML files");
+						Logger.error(this,e.getMessage(),e);
+					}
+
+					try {
+						_xstream.toXML(_list, _bout);
+					} finally {
+						CloseUtils.closeQuietly(_bout);
+					}
+
+					_list = null;
+					_dh = null;
+					_bout = null;
+
+				}
+				Logger.info(this, "writing : " + total + " records for " + clazz.getName());
+			}
+
+			/* Run Liferay's Tables */
+			/* Companies */
+			_list = PublicCompanyFactory.getCompanies();
+			List<Company> companies = new ArrayList<Company>(_list);
+			_xstream = new XStream(new DomDriver());
+			_writing = new File(backupTempFilePath + File.separator +  Company.class.getName() + ".xml");
+			_bout = new BufferedOutputStream(Files.newOutputStream(_writing.toPath()));
+			try {
+				_xstream.toXML(_list, _bout);
+			} finally {
+				CloseUtils.closeQuietly(_bout);
+			}
+			_list = null;
+			_bout = null;
+
+			/* Users */
+			_list = APILocator.getUserAPI().findAllUsers();
+			_list.add(APILocator.getUserAPI().getDefaultUser());
+			_xstream = new XStream(new DomDriver());
+			_writing = new File(backupTempFilePath + File.separator +  User.class.getName() + ".xml");
+			_bout = new BufferedOutputStream(Files.newOutputStream(_writing.toPath()));
+			try {
+				_xstream.toXML(_list, _bout);
+			} finally {
+				CloseUtils.closeQuietly(_bout);
+			}
+			_list = null;
+			_bout = null;
+
+			DotConnect dc = new DotConnect();
+
+			/* counter */
+			dc.setSQL("select * from counter");
+			_list = dc.getResults();

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "getResults"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)